packages/cyberstorm/src/components/componentStyles/DropDown.module.css

```diff
@@ -0,0 +1,35 @@
+.root {
```

Comment:
Adding here as there's no relevant class to attach this to.

While removing the background-color from `MenuItem` had the effect of nullifying the visual z-index issue with the `:focus-visible` -ring, the problem still persists with any custom dropdown items for `DropDown` which have a background-color. However, this is easily fixed by setting a bigger z-index for the item being hovered. This can be targeted with the data-attribute Radix sets automatically, `[data-highlighted]`.

The question of what to do to the ring itself in the long run is another question, but for now this solutions is a good exercise on the matter.